pkg/xtrabackup/xtrabackup.go

```diff
@@ -273,14 +273,6 @@ func deleteContainer(image string, conf *BackupConfig, storage *apiv1alpha1.Back
 				Name:      apiv1alpha1.BinVolumeName,
 				MountPath: apiv1alpha1.BinVolumePath,
 			},
-			{
```

Comment:
@egegunes what do you think about it? We do not use it now, so we can remove it. Then we will see